Firebase/Auth/Source/RPCs/FIRVerifyPhoneNumberRequest.h

```diff
@@ -58,21 +63,25 @@ NS_ASSUME_NONNULL_BEGIN
     @brief Designated initializer.
     @param temporaryProof The temporary proof sent by the backed.
     @param phoneNumber The phone number associated with the credential to be signed in.
+    @param operation Indicates what operation triggered the verify phone number request.
```

Comment:
If you go with the enum route it might better to use enums all the way until the we fill the request body.